### PR TITLE
Document format_date method [ci skip]

### PR DESCRIPTION
**Pull request by jwworth**

### CODE REVIEW

File: actionview/lib/action_view/helpers/tags/week_field.rb [hunk 1]
```diff
@@ -4,9 +4,11 @@ module Tags # :nodoc:
       class WeekField < DatetimeField # :nodoc:
         private
 
-          def format_date(value)
-            value.try(:strftime, "%Y-W%W")
-          end
+        # Returns _value_ in "2015-W41" format if _value_ responds to strftime;
+        # otherwise returns nil.
+        def format_date(value)
```
Comment (arthurnn) [inline actionview/lib/action_view/helpers/tags/week_field.rb]: I dont think we wanna change the indentation here.

Comment (jwworth) [inline actionview/lib/action_view/helpers/tags/week_field.rb]: @arthurnn okay; will make that change.

**Comment by pixeltrix** [comment]
@jwworth since this method is private and the implementation is clear I don't think adding this comment would be of any benefit. Thanks for your PR anyway though - sorry it didn't get merged.

**Comment by jwworth** [comment]
@pixeltrix that makes sense. Thanks for the feedback.

**Pull request closed without merging**